build_scripts/build_linux_rpm-2-installer.sh

```diff
@@ -74,64 +70,45 @@ fpm -s dir -t rpm \
 cp -r dist/daemon ../chia-blockchain-gui/packages/gui
 
 # Change to the gui package
-cd ../chia-blockchain-gui/packages/gui || exit
+cd ../chia-blockchain-gui/packages/gui
```

Comment:
should retain all `|| exit` on any `cd` commands, see comment above

Reply:
Updated.